oauth2client/flask_util.py

```diff
@@ -15,10 +15,17 @@
 """Utilities for the Flask web framework
 
 Provides a Flask extension that makes using OAuth2 web server flow easier.
-The extension includes views that handle the entire auth flow and a @required
-decorator to automatically ensure that user credentials are available.
+The extension includes views that handle the entire auth flow and a
+``@required`` decorator to automatically ensure that user credentials are
+available.
 
-To configure::
+
+Configuration
+=============
+
+To configure, you'll need a set of OAuth2 client ID from the
+`Google Developer's Console <https://console.developers.google.com/project/_/\
```

Comment:
Sidenote: Sphinx autodoc gets strings by importing the modules rather than scanning them, so the Python interpreter gets a chance to lex/parse the docstrings first. The usual escapes all happily apply (e.g. the newline here).